sonar-application/src/main/java/org/sonar/application/ClusterProperties.java

```diff
@@ -0,0 +1,162 @@
+/*
+ * SonarQube
+ * Copyright (C) 2009-2017 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+package org.sonar.application;
+
+import org.apache.commons.lang3.StringUtils;
+import org.sonar.process.Props;
+
+/**
+ * Properties of the cluster configuration
+ */
+final class ClusterProperties {
+
+  enum Parameter {
+    ENABLED("sonar.cluster.enabled", Boolean.FALSE.toString()),
+    MEMBERS("sonar.cluster.members", ""),
+    PORT("sonar.cluster.port", Integer.toString(9003)),
+    PORT_AUTOINCREMENT("sonar.cluster.port_autoincrement", Boolean.FALSE.toString()),
+    INTERFACES("sonar.cluster.interfaces", ""),
+    NAME("sonar.cluster.name", ""),
+    LOG_LEVEL("sonar.cluster.log.level", "WARN");
```

Comment:
this property should follow the standard patter for naming log levels. As such, it should be named "sonar.log.level.app.hazelcast'.

I'm not sure this property should be kept in the long run though, we'll see (we got rid of all such properties over time)